jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -3,332 +3,131 @@ layout: classic-docs
-
-In 2.0, jobs are broken into granular steps. You can compose these steps within a job at your discretion. This gives you greater flexibility to run your build the way you want.
-
-To learn more, please see the [Configuration Reference]({{ site.baseurl }}/2.0/configuration-reference/).
-
-### Custom Build Image
-In 1.0, you’re restricted to the build image CircleCI provides. In Linux builds, there are 2 images you can use: Ubuntu 12.04 and 14.04. While these images come with many languages and tools pre-installed, it’s frustrating if you need a version of a service or dependency that isn’t included.
-
-Maybe you want to use a different version of MySQL than the one included in either default image. Installing another version adds time and complexity to your builds.
-
-In 2.0, we support almost all public Docker images. You can also create a custom image and run jobs on that. You can even compose multiple images together (like MySQL 5.7 + Redis 3.2) and run jobs on them as if they were a single image.
-
-To learn more, please see the [Project Walkthrough]( {{ site.baseurl }}/2.0/project-walkthrough/).
-
-----
-
-# Configuration Equivalents
-CircleCI 2.0 introduces a completely new syntax in `.circleci/config.yml`. This article will help you convert your 1.0 `circle.yml` to a 2.0 `config.yml`.
-
-
-## Machine
-
-### Timezone
-
-**1.0**
-
-```YAML
-machine:
-  timezone: Europe/Dublin
-```
-
-**2.0**
-
-You can set the timezone in Docker images with the `TZ` environment variable. The value should be something like: `TZ="/usr/share/zoneinfo/America/Los_Angeles"`. A full list of available timezone options is [available on Wikipedia](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
-
-In your `.circleci/config.yml` it would look like:
+CircleCI 2.0 introduces new keys in `.circleci/config.yml`. This article will help you search and replace your 1.0 keys with 2.0 keys.
+
+## Steps to Configure Required 2.0 Keys
+
+1. Copy your existing `circle.yml` file into a new directory called `.circle` at the root of your project repository.
+
+2. Rename `.circle/circle.yml` to `.circle/config.yml`.
+
+3. Add `version: 2` to the top of the `.circle/config.yml` file.
+
+4. Add the following two lines to your `config.yml` file, after the verison line. If your configuration includes `machine:`, replace `machine:` with the following two lines, nesting all of the following sections under `build`.
+     ```
+     jobs:
+       build:
+     ```
+5. Add the language and version to your configuration using either the `docker:` and `- image:` keys in the example or by setting `machine: true`. If your configuration includes language and version as shown for `ruby:` below, replace it as shown.
+     ```
+       ruby:
+         version: 2.3
+     ```
+     Replace with the following two lines:
+     ```
+         docker:
+           - image: ruby:2.3
+     ```
+     Or
+
+     ```
+    machine: true
```

Comment:
not sure we should be suggesting to use `machine: true` without context. People should default to `docker` unless they really need the machine executor.